### ISSUE TITLE
False positive when using spotbugs annotations

### ISSUE THREAD

**Issue by vbrandl** [nebula-plugins/gradle-lint-plugin#239]
The [spotbugs migration guide](https://spotbugs.readthedocs.io/en/latest/migration.html#com-google-code-findbugs-annotations) suggests to replace `com.google.code.findbugs:annotations` with `net.jcip:jcip-annotations:1.0` and `com.github.spotbugs:spotbugs-annotations:4.0.0-beta1`.

When doing so on a project, that uses spotbugs, the lint plugin fails with

```
undeclared-dependency              one or more classes in com.google.code.findbugs:annotations:2.0.1 are required by your code directly
unused-dependency                  one or more classes in com.google.code.findbugs:annotations:2.0.1 are required by your code directly
```
